paasta_itests/steps/bounces_steps.py

```diff
@@ -36,6 +36,15 @@ def which_id(context, which):
 
 @given(u'a new {state} app to be deployed, with bounce strategy "{bounce_method}" and drain method "{drain_method}"')
 def given_a_new_app_to_be_deployed(context, state, bounce_method, drain_method):
+    given_a_new_app_to_be_deployed_constraints(context, state, bounce_method, drain_method, str(None))
```

Comment:
Forgot about this bit. This value gets eval'ed further down. The reason for this is that behave doesn't support an array type parameter in a step definition. So we read in the array as a string and eval it back to the proper type. The `str(None)` is really `eval(str(None))`. `eval()` gets upset if you simply pass it an empty string.